src/main/java/io/asyncer/r2dbc/mysql/MySqlConnection.java

```diff
@@ -92,6 +94,19 @@ public final class MySqlConnection implements Connection, ConnectionState {
         }
     };
 
+    private static final BiConsumer<ServerMessage, SynchronousSink<Void>> RESET = (message, sink) -> {
+        if (message instanceof ErrorMessage) {
+            ErrorMessage msg = (ErrorMessage) message;
+            logger.warn("Reset connection failed: [{}] [{}] {}", msg.getCode(), msg.getSqlState(),
+                msg.getMessage());
+            sink.complete();
+        } else if (message instanceof CompleteMessage && ((CompleteMessage) message).isDone()) {
+            sink.complete();
+        } else {
+            ReferenceCountUtil.safeRelease(message);
```

Comment:
Do we forget to call `sink#complete()` after this line? Otherwise, if it enters this condition, it will never contimue. If I have overlooked anything, please feel free to correct me.

Reply:
This command should only respond Ok/Err packets, so we wait for Ok/Err packets, and discard other packets at this line.

See also [`MySqlConnection.PING`](https://github.com/asyncer-io/r2dbc-mysql/blob/trunk/src/main/java/io/asyncer/r2dbc/mysql/MySqlConnection.java#L80)

Reply:
Make sense.